app/nw/scripts/build-app.mjs

```diff
@@ -23,6 +23,10 @@ const manifest = {
         'http://localhost/*',
         'https://haruneko.pages.dev/*'
     ],
+    // See: https://peter.sh/experiments/chromium-command-line-switches/
+    'chromium-args': [
+        '--ignore-certificate-errors',
```

Comment:
Arent there security implications ? Should we leave it like that ?

Reply:
It is not safe, but required since TMO has an invalid certificate which must be manually allowed in browser. Since users (hopefully) don't use HaruNeko for banking or other important account logins, it can be considered as very low risk.

Reply:
In the meantime TMO fixed their untrusted certificate.
Removed permission for untrusted certificates in c2df72d3